endercore/src/main/java/com/enderio/core/common/lang/EnderCoreLang.java

```diff
@@ -0,0 +1,16 @@
+package com.enderio.core.common.lang;
+
+import com.enderio.core.EnderCore;
+import com.tterrag.registrate.Registrate;
+import net.minecraft.network.chat.TranslatableComponent;
+
+public class EnderCoreLang {
```

Comment:
I'd rather us avoid using Registrate in EnderCore; could we manually write this in the langfile instead (similar to how it is done in #40

Reply:
To add - registrate is only present in endercore right now due to the gradle project. EnderCore will move to its own git once we're not developing it so fast.